lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.h

```diff
@@ -481,7 +482,7 @@ class SwiftASTContext : public TypeSystemSwift {
                           uint32_t bufferID = UINT32_MAX,
                           uint32_t first_line = 0,
                           uint32_t last_line = UINT32_MAX) const;
-    bool HasErrors() const;
+    std::optional<ErrorKind> GetErrorKindIfAny() const;
```

Comment:
what do you think about just `GetErrorKind`? Does the return type conveys the "IfAny"?

Reply:
I think that `GetErrorKindIfAny` conveys that there's a possibility that there's no error, especially if callers use `auto` or it's called inside the condition of an if statement.